app/routes/__index/domains/index.tsx

```diff
@@ -31,6 +32,11 @@ export default function DomainsIndexRoute() {
         been the industry's standard dummy text ever since the 1500s, when an unknown printer took a
         galley of type and scrambled it to make a type specimen book.
       </Text>
+      <Flex justifyContent="flex-end">
+        <Link to="/domains/new">
+          <Button rightIcon={<AddIcon boxSize={3} />}>Create new domain</Button>
```

Comment:
I wonder if `Domain` (capital) is better?  Not sure.

Reply:
![image](https://user-images.githubusercontent.com/32075241/218796368-5ca73a4c-4049-416f-84f1-1184db8c0695.png)

Reply:
I don't have a hard opinion on that, if you guts want I can change it to capital

Reply:
Same, we should just set a standard and stick with it.